drizzlepac/haputils/deconvolve_utils.py

```diff
@@ -915,10 +920,25 @@ def find_point_sources(drzname, data=None, mask=None,
     clean_psfs = True if not diagnostic_mode else False
 
     drzpsfname = convert_library_psf(calname, drzname, psfnames,
-                                     pixfrac=0.8,
+                                     pixfrac=1.5,
                                      clean_psfs=clean_psfs)
     drzpsf = fits.getdata(drzpsfname)
-    psf_fwhm = amutils.find_fwhm(drzpsf, 2.0)
+    # try to measure just the core of the PSF
+    # This will be a lot less likely to result in invalid/impossible FWHM values
+    yc, xc = np.where(drzpsf == drzpsf.max())[0]
+    psf_core = drzpsf[yc - box_size: yc + box_size, xc - box_size: yc - box_size]
+    psf_fwhm = amutils.find_fwhm(psf_core, def_fwhm)
```

Comment:
I just want to make sure line 929 is correct in terms of the limits of the box being extracted (second dimension).

Reply:
Good catch!  Fat fingers and all that got the better of me there.  Fixed in 8030dc61